http/logical.go

```diff
@@ -430,6 +430,7 @@ func respondLogical(core *vault.Core, w http.ResponseWriter, r *http.Request, re
 					CreationPath:    resp.WrapInfo.CreationPath,
 					WrappedAccessor: resp.WrapInfo.WrappedAccessor,
 				},
+				MountType: resp.MountType,
```

Comment:
Isn't this only going to be populated for wrapped requests, based on the `if` it's in?

Reply:
Yeah, but the only other way it's populated is via the 'else' block. In other words, without this, it won't ever be populated for wrapped requests

Reply:
Oh, I see, it's for a different use case... I guess the router based change doesn't cover that?  Can we add a test for it?

Reply:
Hmm, I couldn't find a test for this and on second thought, I don't know that it should actually be there to begin with, as we are _only_ returning WrapInfo and nothing else, so it'd be strange if this were the first non-WrapInfo thing we return. I'll remove it from this path.